core/cap-0040.md

```diff
@@ -163,10 +167,17 @@ This proposal provides a primitive that makes it possible to construct a set
 of transactions where all transactions can be authorized and submitted if the
 first transaction is authorized and submitted.
 
-This proposal makes this possible when the signer is used with the
-`extraSigners` precondition introduced in [CAP-21], and does so without
-introducing any new state to be stored on the ledger, allowing the signing
-dependency to exist without creating ledger entries.
+This proposal makes this possible since a Stellar transaction hash can be
+specified as the payload of an ed25519 signed payload signer. A Stellar
+transaction hash is the SHA-256 hash of the `TransactionSignaturePayload`
+containing the Stellar transaction. A signature for the signer will also be an
+ed25519 signature of the other Stellar transaction that could be attached to the
+other transaction.
```

Comment:
> A signature for the signer will also be an ed25519 signature of the other Stellar transaction that could be attached to the other transaction.

I _think_ I follow this, but "other transaction" seems overloaded here? Is this saying that a payload signature here could be identical to another transaction's "normal" signature?

Reply:
Ah, thanks. Addressed.

Reply:
The signatures won't be identical, but one can be ported to the other without any secret information.

Reply:
They aren't identical because the hint will be different.